go.mod

```diff
@@ -62,6 +62,7 @@ require (
 	github.com/go-openapi/jsonpointer v0.19.5 // indirect
 	github.com/go-openapi/swag v0.21.1 // indirect
 	github.com/go-sql-driver/mysql v1.6.0 // indirect
+	github.com/go-test/deep v1.0.8 // indirect
```

Comment:
Do we have any company restrictions to import 3rd party packages?

Reply:
It is used just for tests. So should be any proplems